flash/image/detection/data.py

```diff
@@ -407,3 +412,80 @@ def from_fiftyone(
             ),
             **data_module_kwargs,
         )
+
+    @classmethod
+    def from_folders(
+        cls,
+        predict_folder: Optional[str] = None,
+        predict_transform: Optional[Dict[str, Callable]] = None,
+        image_size: Tuple[int, int] = (128, 128),
+        **data_module_kwargs: Any,
+    ) -> "DataModule":
+        """Creates a :class:`~flash.core.data.data_module.DataModule` object from the given folders using the
+        :class:`~flash.core.data.io.input.Input` of name
+        :attr:`~flash.core.data.io.input.InputFormat.FOLDERS`
+        from the passed or constructed :class:`~flash.core.data.io.input_transform.InputTransform`.
+
+        Args:
+            predict_folder: The folder containing the predict data.
+            predict_transform: The dictionary of transforms to use during predicting which maps
+            data_module_kwargs: The keywords arguments for creating the datamodule.
+
+        Returns:
+            The constructed data module.
+        """
+        return cls(
+            None,
+            None,
+            None,
+            IceVisionInput(RunningStage.PREDICTING, predict_folder),
+            input_transform=cls.input_transform_cls(
+                None,
+                None,
+                None,
+                predict_transform,
```

Comment:
Could use kwargs here:
```suggestion
            predict_dataset=IceVisionInput(RunningStage.PREDICTING, predict_folder),
            input_transform=cls.input_transform_cls(
                predict_transform=predict_transform,
```

Reply:
Done !